code/game/gamemodes/game_mode.dm

```diff
@@ -91,6 +85,17 @@
 /datum/game_mode/process()
 	return 0
 
+// I wonder what this could do guessing by the name
+/datum/game_mode/proc/set_mode_in_db()
+	if(SSticker && SSticker.mode && SSdbcore.IsConnected())
+		var/datum/db_query/query_round_game_mode = SSdbcore.NewQuery("UPDATE [format_table_name("round")] SET game_mode=:gm WHERE id=:rid", list(
+			"gm" = SSticker.mode,
```

Comment:
https://github.com/ParadiseSS13/Paradise/blob/9480211187438f8e7a54d3d87b838531efe9af2c/code/controllers/subsystem/ticker.dm#L22-L23

`SSticker.mode` is a datum. How does that work?

Additionally, any reason you get the information from the ticker instead of the mode (`src`) itself?

Reply:
It takes the `name` var, but I will force specify it. As for why I do the ticker lookup, sometimes with forced secret modes things can break, so this is a surefire way that it grabs the right thing.